### ISSUE TITLE
Add week number?

### ISSUE THREAD

**Issue by roarnyg** [knopkem/yaps#3]
What about adding week numbers, so we can structure the file names and folders even more? I would like only 52 folders for each year.

**Comment by knopkem** [comment]
Yes, this might be a good option, will see if I find the time to add this.

**Comment by knopkem** [comment]
week number added, let me know if there are any issues.
